OpenQA/Qemu/DriveDevice.pm

```diff
@@ -98,6 +112,9 @@ sub gen_cmdline ($self) {
         $self->_push_ifdef(\@params, 'bootindex=', $self->bootindex) if (!$path->id || $path->id eq 'path0');
         $self->_push_ifdef(\@params, 'serial=', $self->serial);
         $self->_push_ifdef(\@params, 'num_queues=', $self->num_queues) if (($self->num_queues // -1) != -1);
+        # logical_block_size=4096,physical_block_size=4096
+        $self->(_push_ifdef(\@params, 'logical_block_size=', $self->drive->logical_block_size));
```

Comment:
if you want to call `$self->drive->logical_block_size`, then `logical_block_size` has to be added to `OpenQA::Qemu::BlockDev`

Reply:
@perlpunk I'm having the feeling that logical and physical have to be completely moved to BlockDev - Gotta understand a bit the relationship between the overlays and the backing device (and how it may affect other parts, if at all)